src/utils/errors/errorHandling.ts

```diff
@@ -70,8 +72,10 @@ const getErrorSignature = (error: any): string => {
   } catch {
     console.error(`Failing to get error signature. ${error}`);
     const sentryTracker = getSentryTracker();
-    sentryTracker.captureException(error);
-    sentryTracker.captureMessage(`Failing to get error signature. ${error}`);
+    if (sentryTracker) {
```

Comment:
in what cases can sentryTracker be optional? Please explain

Reply:
We realised that Sentry Browser Client didn't work when we tried to use SDK in google cloud back-end. We can investigate more options of clients.